src/client/datascience/commands/commandRegistry.ts

```diff
@@ -487,6 +488,10 @@ export class CommandRegistry implements IDisposable {
         env.openExternal(Uri.parse(`https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter`));
     }
 
+    // Open up our variable viewer using the command that VS Code provides for this
+    private async openVariableView(): Promise<void> {
+        this.commandManager.executeCommand('jupyterViewVariables.focus');
```

Comment:
I actually took a bit of time working on this looking at how VS Code handles view / opening commands. I was struggling a bit as the View: Open View command didn't take a parameter, it just always pops up a selector. Was expecting to use that command with my view name. Digging into the code I found that VS code actually creates focus commands for all views that are contributed, which I didn't realize. I'm using that command to do the focus, but I still want our own command so that we can find it in the menus under the Jupyter titles instead of in the views section.

Reply:
I think we need an away for `executeCommand`, wait for it to complete.

Reply:
If we want to fire & forget, then we should make `openVariablesView` a sync function.
Else someone calling this in the future might make the wrong assumption.